src/test/ensemble_mapping_config_tests.cpp

```diff
@@ -40,7 +40,7 @@ class PipelineWithInputOutputNameMappedModel : public ::testing::Test {
             ::testing::UnitTest::GetInstance()->current_test_info();
 
         const std::string directoryName = std::string(test_info->test_suite_name());
-        directoryPath = "/tmp" + directoryName;
+        directoryPath = "/tmp/" + directoryName;
```

Comment:
Why this change? I mean tests were silently passing because of that?

Reply:
Tests were passing, but files were not created in /tmp directory. Purpose of this test suite was to place it everything under /tmp.